cookbook/bundles/tests.rst

```diff
@@ -0,0 +1,260 @@
+.. index::
+   single: Bundle; Testing third party bundles
+
+Writing Tests for Third Party Bundles
+=====================================
+
+The Standard Edition's front controller looks like this::
+
+    // web/app.php
+    use Symfony\Component\HttpFoundation\Request;
+
+    require_once __DIR__.'/../vendor/autoload.php';
+    require_once __DIR__.'/../app/AppKernel.php';
+
+    // Create the request from PHP variables super globals such as $_GET, $_POST, $_SERVER, etc
+    $request = Request::createFromGlobals();
+
+    // The application handles the Request
+    $kernel = new AppKernel('prod', false);
+    $response = $kernel->handle($request);
+
+    // Dump the Response's headers and print the Response's content
+    $response->send();
+
+In this example, Symfony is used as an application that mimicks the HTTP protocol,
+which means that it is possible to write transversal tests (System Tests) by building
+"manually" a ``Request``, giving it to our application and then check the returned ``Response``.
+
+This article provides examples on how to write such tests with third party bundles.
+
+Creating a Minimal AppKernel
+----------------------------
+
+Third party bundles cannot be ran standalone, they need an application.
+It is possible to create a minimalistic ``AppKernel`` inside the bundle for testing
+and showcase purposes, instead of:
+
+1. creating a separate application
+2. install the bundle in it
+3. then check how things turn out
```

Comment:
reSt uses `#.` for numbered lists. Also, we normally capitialize the first word of each list item and we end it with a semi-colon (except from the last one, which ends with a full stop)